container/navigation.go

```diff
@@ -0,0 +1,118 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Navigation container is used to provide your application with a control bar and an area for content objects.
+// Objects can be any CanvasObject, and only the most recent one will be visible.
+//
+// Since: 2.6
+type Navigation struct {
+	widget.BaseWidget
+	control *fyne.Container
+	stack   *fyne.Container
+	button  *widget.Button
+	label   *widget.Label
+	title   string
+	titles  []string
+}
+
+// NewNavigation creates a new navigation container allowing to pass in one or more objects.
+//
+// Since: 2.6
+func NewNavigation(s string, objs ...fyne.CanvasObject) *Navigation {
+	button := widget.NewButtonWithIcon("", theme.NavigateBackIcon(), nil)
+	label := widget.NewLabelWithStyle(s, fyne.TextAlignCenter, fyne.TextStyle{Bold: true})
+	titles := make([]string, len(objs))
+	for n := 0; n < len(objs); n++ {
+		titles[n] = s
+	}
+	nav := &Navigation{
+		button:  button,
+		control: NewStack(NewHBox(button), label),
+		stack:   NewStack(objs...),
+		label:   label,
+		title:   s,
+		titles:  titles,
+	}
+	nav.ExtendBaseWidget(nav)
+	nav.button.OnTapped = func() {
+		nav.Pop()
+	}
```

Comment:
```suggestion
	nav.button.OnTapped = nav.Pop
```

Reply:
Different function signature, forgot to explicitly ignore the return value. Still raising the API question, should Pop() return the object?